eventsProject/trendingEvents.php

```diff
@@ -13,19 +13,17 @@
 
 $allButTopThree=getAllEventsButTopThree();
 $topThree=getTopThreeEvents();
-$event1=getOneEvent($topThree[0]['eventID']);
-$event1ID=$event1['eventID'];
-$event2=getOneEvent($topThree[1]['eventID']);
-$event2ID=$event2['eventID'];
-$event3=getOneEvent($topThree[2]['eventID']);
-$event3ID=$event3['eventID'];
+
```

Comment:
It might not seem like a problem with your dev server, but it's normally best to make the fewest number of database queries possible. This is because on a real web site, the database is normally on a totally different server from the php server. So even though the queries are fast, the time it takes going back and forth between PHP and Mysql can add up. This is called latency, and it's something to be aware of anytime you write code that runs on a network (including but not limited to the internet).

So how about instead, you get all the events you need, loop through that and pull out the top three, and don't call `getOneEvent` for any of those? That way you could do this whole thing in one query.